src/components/NcHeaderMenu/header-menu__trigger.scss

```diff
@@ -0,0 +1,33 @@
+.header-menu {
+	position: relative;
+	width: var(--header-height);
+	height: var(--header-height);
+
+	#{&}__trigger {
+		width: 100% !important;
+		height: var(--header-height);
+		opacity: .85;
+
+		// header is filled with primary or image background
+		filter: none !important;
+		color: var(--color-background-plain-text, var(--color-primary-text)) !important;
+
+		&:focus-visible {
+			outline: none !important;
+			box-shadow: none !important;
+		}
+	}
+
+	&--opened &__trigger,
+	&__trigger:hover,
+	&__trigger:focus,
+	&__trigger:active {
+		opacity: 1;
+	}
+}
+
+@media only screen and (max-width: $breakpoint-small-mobile) {
```

Comment:
NcButton is still considered `large`, and  might cause horizontal scrolling on small viewports (see Contacts Menu).

Can we apply something like this?

[fix_NcHeader.patch](https://github.com/user-attachments/files/16905648/fix_NcHeader.patch)

Reply:
> NcButton is still considered `large`

Yes because I did not yet rebase after #6047

Reply:
Aye aye